dnd.sty

```diff
@@ -35,6 +36,8 @@
   \PackageWarning{dnd}{#1 is deprecated and will be removed in version #2.\IfNoValueF{#3}{ #3}}
 }
 
+\dnd@deprecate{Package `hang`}{0.7}[Require package `hang` in your document directly.]
```

Comment:
Why is this included?

Reply:
Previously we used `hangingpar` (from the `hang` package) for the hanging indents in stat blocks.

I originally removed the package include entirely, but I realized some people may have used `hangingpar` in their own documents without declaring the dependency directly.

Reply:
That change was introduced after 0.6.0 was released. I think it is okay to just remove it since we have not had a formal release with that included.

Reply:
:+1: